src/rprblender/nodes/node_parser.py

```diff
@@ -244,13 +244,33 @@ def export(self):
         """
         pass
 
+    def export_muted(self):
+        # export as a muted node
+        # pass through first instance of same socket type
+        matching_incoming_socket = next((input for input in self.node.inputs
+                    if isinstance(input, type(self.socket_out))), None)
+        if matching_incoming_socket:
+            if matching_incoming_socket.is_linked:
+                # pass through first incoming link if there is one
+                return self.get_input_link(matching_incoming_socket.name)
+            else:
+                # return black
+                return NodeItem(self.rpr_context, (0.0, 0.0, 0.0, 0.0))
```

Comment:
BaseNodeParser doesn't work with NodeItem, therefore here we should return just (0.0, 0.0, 0.0, 0.0).
Also export_muted() function should be overrided in NodeItem in the same way as get_input_link().